docs/nixos-with-flakes/other-useful-tips.md

```diff
@@ -9,8 +9,8 @@ However, by default, NixOS places the configuration in `/etc/nixos`, which requi
 For example, you can place your flake in `~/nixos-config` and create a symbolic link in `/etc/nixos` as follows:
 
 ```shell
-sudo mv /etc/nixos /etc/nixos.bak  # Backup the original configuration
```

Comment:
To create a symlink to `/etc/nixos`, We have to move it to another place first, so I think `mv` should still be used here.